submissions/Harnytskyi/friends-app/js/script.js

```diff
@@ -0,0 +1,163 @@
+const GRID = document.getElementById("allFriends");
+const CHECKBOX_MAN = document.getElementById("gender-male");
+const CHECKBOX_WOMAN = document.getElementById("gender-female");
+const SORT_ALPHABET_ASCEND = document.getElementById("a-z");
+const SORT_ALPHABET_DESCEND = document.getElementById("z-a");
+const SEARCH_INPUT = document.getElementById('search');
+const SORT_AGE_ASCEND = document.getElementById("0-9");
+const SORT_AGE_DESCEND = document.getElementById("9-0");
+const FILTER_SORT = document.getElementById("filter-sort");
+const FILTER_GENDER = document.getElementById("filter-gender");
```

Comment:
Don’t overuse global variables, keep data that you are using closest as possible to the place where you using it.